src/pages/Dashboard/components/ProfileAvatar/AvatarSelectModal.tsx

```diff
@@ -27,19 +31,20 @@ import type { NftItemWithCollection } from 'state/apis/nft/types'
 import { selectWalletAccountIds, selectWalletId } from 'state/slices/common-selectors'
 import { useAppDispatch, useAppSelector } from 'state/store'
 
-import { AvatarRadio } from './AvatarRadio'
+import { NftRow } from './NftRow'
 
 type AvatarSelectModalProps = Pick<ModalProps, 'isOpen'> &
   Pick<ModalProps, 'onClose'> & { walletImage: string }
 
 export const AvatarSelectModal: React.FC<AvatarSelectModalProps> = props => {
-  const [selected, setSelected] = useState<AssetId | null>(null)
+  const [selected, setSelected] = useState<string | null>(null)
```

Comment:
As discussed, this is correct and actually a possible string now that the default asset image (not an AssetId) is part of the possible values of `selected`